File: output/stringConcat.java
public class stringConcat {
public static void main(String args[]) {
	String s = ("foo", "bar");
	System.out.println(s);	String s1 = "foo";
	String s2 = ("bar", s1);
	System.out.println(s2);	System.out.println(("foo", ("bar", "baz")));
}

}

File: output/c/stringConcat.c
#include "concat.h"
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <stdbool.h>

int main() {
	char * s = concat("foo", "bar");
	printf("%s", s);
	char * s1 = "foo";
	char * s2 = concat("bar", s1);
	printf("%s", s2);
	printf("%s", concat("foo", concat("bar", "baz")));

	return 0;
}
